### PR TITLE
change code font to Ubunto Mono

### PR DESCRIPTION
**Pull request by behoppe**
@cleiserson @neboat @ailiop I diagnosed the "small code font" problem, and with this PR propose to standardize our code font as Ubuntu Mono. We want to use the same font in all our figures etc., so if you don't like Ubuntu Mono, please let me know.

My rationale for Ubuntu Mono is that we already use Ubuntu as the headline font. So using Ubuntu Mono reinforces our headline aesthetic. (The regular body text is Roboto btw.) Here is a [comparison of Ubuntu Mono with Consolas](http://www.identifont.com/differences?first=Ubuntu+Mono&second=Consolas), and a couple other comments about it [here](https://www.slant.co/versus/226/230/~ubuntu-mono_vs_consolas) and [here](https://spin.atomicobject.com/2016/07/11/programming-fonts/).

On an accidentally related note, I discovered that after I installed GoogleFont Consolas on my local instance, then I too got the "small code font" problem that we discussed, until I resolved some `font-size` issues in `index.css` and `prism.css` that are part of this PR.

So bottom line, this PR makes Ubuntu Mono the code font and it fixes `font-size` problems to make the code font match the regular.

### CODE REVIEW

**Comment by behoppe** [comment]
I went ahead and merged this, but I am still happy to take feedback on Ubuntu Mono and the other changes here.
